client/me/purchases/confirm-cancel-domain/style.scss

```diff
@@ -0,0 +1,25 @@
+.confirm-cancel-domain {
+	// the following is a hack so the dropdown selector is not cropped when it overflows its container
```

Comment:
I haven't found a nice way to fix this issue yet. If we remove this line the dropdown is cropped as shown here 
![image](https://cloud.githubusercontent.com/assets/230230/14235682/65e6a478-fa05-11e5-8bd1-3c990c5847fb.png)
A simple solution would be to use a select element instead of the `SelectDropdown` component.